backend/pages/utils/advanced_processing.py

```
@@ -0,0 +1,136 @@
+    "Other Income":                             {"type": "relative", "sign": 1},
+    "Other Expenses":                           {"type": "relative", "sign": -1},
+    "Earnings Before Interest and Taxes":       {"type": "total",    "sign": 1},
+    "Interest Income":                          {"type": "relative", "sign": 1},
+    "Interest Charges":                         {"type": "relative", "sign": -1},
+    "Income Before Taxes":                      {"type": "total",    "sign": 1},
+    "Loss Carry Forward":                       {"type": "relative", "sign": -1},
+    "Taxable Income":                           {"ignore":True},
+    "Income Taxes":                             {"type": "relative", "sign": -1},
+    "Net Income":                               {"type": "total",    "sign": 1},
+    "Earnings per Share":                       {"ignore": True},
+    "Gross Profit":                             {"ignore": True},
```

Ruff (0.8.2)

Dictionary key literal `"Gross Profit"` repeated

Remove repeated key literal `"Gross Profit"`

(F601)